pkg/service/mockrecord/mockrecord.go

```diff
@@ -58,10 +67,21 @@ func (s *mockRecorder) MockRecord(path string, pid uint32, mockName string) {
 	signal.Notify(stopper, syscall.SIGINT, syscall.SIGTERM)
 
 	fmt.Printf(Emoji+"Received signal:%v\n", <-stopper)
+	mocksRecorded := make(map[string]int)
+	tcsMocks, configMocks, err := ys.ReadMocks(path)
+	if err != nil{
+		s.logger.Debug("Failed to read mocks")
+	}
+	tcsMocks = append(tcsMocks, configMocks...)
+	for _, mock := range tcsMocks {
+		mocksRecorded[string(mock.Kind)] ++
+	}
 
 	s.logger.Info("Received signal, initiating graceful shutdown...")
 
 	// Shutdown other resources
 	loadedHooks.Stop(true, nil)
+	//Call the telemetry events.
+	tele.RecordedMock(mocksRecorded)
```

Comment:
In MockTest you did this before loadedHooks. Follow one convention.

Reply:
done